shepherd_bms/include/compute.h

```diff
@@ -2,5 +2,59 @@
 #define COMPUTE_H
 
 #include "datastructs.h"
+#include "nerduino.h"
 
+using namespace std;
+
+class ComputeInterface
+{
+    private:
+        uint8_t fanSpeed;
+
+        /**
+         * @todo These might need to be changed depending on the charging ticket
+         */
+        const uint8_t startChargingMsg[];
```

Comment:
Yeah this is almost certainly not how it works. There is a constant message that is sent with the current/voltage that we want delivered

Reply:
If this is a placeholder anyways though we can leave it for now. But may already want to rename to "chargingCmdMsg" or something

Reply:
Yeah I might change it when I get back and look into what the charger message actually contains